components/table/table-wrapper.js

```diff
@@ -381,6 +391,25 @@ export class TableWrapper extends RtlMixin(PageableMixin(SelectionMixin(LitEleme
 		});
 	}
 
+	_checkSiblingSortableCells(c) {
+		const nodes = Array.from(c.childNodes);
+		const isSortableCell = nodes.some((element) => element.localName === 'd2l-table-col-sort-button');
+
+		if (isSortableCell) {
+			let siblings = 0;
+			nodes.forEach((element) => {
+				if (element.localName) siblings++;
+			});
+
+			c.classList.toggle('d2l-table-header-col-sortable', siblings === 1);
```

Comment:
Is this saying if there is only one item in the cell then toggle this class? If so, I'd recommend a bit of restructuring with this as `siblings` is a bit misleading here.